src/priorityQueue.js

```diff
@@ -0,0 +1,72 @@
+export class Element {
+  constructor(value, priorityInt) {
+    this.data = value
+    this.priority = priorityInt
+  }
+}
+
+export default class PriorityQueue {
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  enqueue(value, priorityInt) {
+    const element = new Element(value, priorityInt)
+    this.elements.push(element)
+    this.size += 1
+  }
+
+  front = () => {
```

Comment:
I recommend using a consistent method declaration style within a class.  I think best practice is to use the form:
```js
front() {
  // etc.
}
```